scripts/chip_heights_format_for_batoid.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+from pathlib import Path
+
+import numpy as np
+from astropy.io import fits
+from astropy.table import Table
+from scipy.spatial import Delaunay
+from sklearn.neighbors import KNeighborsRegressor
+
+
+def process_fits(raw_file: Path, proc_file: Path) -> None:
+    """Process FITS with focal plane height measurements
+    into an Astropy table.
+
+    - Reads all BinTable HDUs whose EXTNAME starts with 'R'.
+    - Normalizes z to zero median over science sensors within
+    317 mm radius.
+    - Removes CWFS offsets (+1.5 mm for SW1, -1.5 mm for SW0).
```

Comment:
This comment is a little confusing (sorry if it was inherited from my code you copied! 🙈 )

It doesn't make it clear if those values are the original offsets, or the deltas we add to remove those offsets (and it's the latter). I'm just worried someone would read the comment without checking the code below and assume +z is the intrafocal direction (i.e. in the direction of SW1).

Maybe change it to
```python
- Removes CWFS offsets (i.e., add +1.5 mm for SW1, -1.5 mm for SW0).
```